Source/WebKit/UIProcess/API/glib/WebKitWebContext.cpp

```diff
@@ -1346,6 +1347,26 @@ void webkit_web_context_set_sandbox_enabled(WebKitWebContext* context, gboolean
     context->priv->processPool->setSandboxEnabled(enabled);
 }
 
+static bool pathIsHomeDirectory(const char* path)
+{
+    GUniquePtr<char> resolvedPath(realpath(path, nullptr));
```

Comment:
Is it a good idea to free with `g_free()` something allocated with system malloc? I guess `g_free()` always ends up calling free nowadays, but still. I think it's better to stack allocate a buffer of `PATH_MAX` and pass it a argument to realpath